functions/commonfn.php

```diff
@@ -334,8 +337,9 @@ function amuRegisterUser(){
         } else if(email_exists($this->user_email)) {
             $newid = __('Error: a user with the user_email address','amulang').' '.$this->user_email.' '.__('already exists. This user was not registered.','amulang');
         } else if($validateEmail == 'yes' && !is_email($this->user_email)) {
-            $newid = __('Error: The user_email provided','amulang').' '.$this->user_email.' '.__('was not valid. This user was not registered.','amulang');
-
+            $newid = __('Error: The user_email provided', 'amulang') . ' ' . $this->user_email . ' ' . __('was not valid. This user was not registered.', 'amulang');
+        } else if(! $this->is_valid_uun($this->user_login)) {
+            $newid = __('Error: username ','amulang').' '.$this->user_login.' '.__('is not a valid Univeristy of Edinburgh uun','amulang');
```

Comment:
Can we change uun to be capitalised? Cheers Richard.

Reply:
done

Reply:
Hey @richeduni. It looks like there has been no commit made. Could you take a second look? Cheers!

Reply:
should be updated now

Reply:
@richeduni I think you are looking at the line above the one I've mentioned. I've reverted the function name to lower case. I was meaning the string on line 342 :-) I've updated this now as was making a change anyways. Cheers

Reply:
thanks